src/index.js

```diff
@@ -53,6 +53,13 @@ class FloodSub extends EventEmitter {
      */
     this.subscriptions = new Set()
 
+    /**
+     * Cache for the last message on seen subscriptions
+     * @type {Map<string, Buffer>}
+     * @private
+     */
+    this._lvc = new Map()
```

Comment:
Yeah need to figure out what's the best caching strategy here. For now I just want to test if this actually solves our problems